src/main/java/org/example/lesson11/task2/ObjectOccurrence.java

```diff
@@ -0,0 +1,43 @@
+package org.example.lesson11.task2;
+
+import lombok.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@NoArgsConstructor
+public class ObjectOccurrence {
+    public void calcOccurrence(List<String> list) {
+        List<Words> wordOccurrences = new ArrayList<>();
+        occurrenceWorker(list, wordOccurrences);
+        resultPrinter(wordOccurrences);
+    }
+
+    public List<Words> findOccurrence(List<String> list) {
+        List<Words> wordOccurrences = new ArrayList<>();
+        occurrenceWorker(list, wordOccurrences);
+        return wordOccurrences;
+    }
+
+    private void resultPrinter(List<Words> wordOccurrences) {
+        for (Words word : wordOccurrences) {
+            System.out.println(word.getName() + ": " + word.getCount());
+        }
+    }
+
+    private void occurrenceWorker(List<String> list, List<Words> wordOccurrences) {
+        for (String word : list) {
+            boolean found = false;
+            for (Words w : wordOccurrences) {
+                if (w.getName().equals(word)) {
+                    w.setCount(w.getCount() + 1);
+                    found = true;
+                    break;
+                }
+            }
+            if (!found) {
```

Comment:
Why if not found?

Reply:
if the word is no longer found, we set its number to 1

Reply:
Very confused construction, honestly, let me reconstruct here, please